provider/k8s/release.go

```diff
@@ -470,6 +470,10 @@ func (p *Provider) releaseTemplateServices(a *structs.App, e structs.Environment
 }
 
 func (p *Provider) releaseTemplateTimer(a *structs.App, e structs.Environment, r *structs.Release, s *manifest.Service, t manifest.Timer) ([]byte, error) {
+	if t.Concurrency != "" {
```

Comment:
@heronrs, the deepsource is complaining about this `strings.Title`. It's saying that is deprecated in go 1.18 but we're not in this version yet. What should I do here? Btw, I didn't like my solution.